contracts/StrategyPrismaConvexFactoryClonable.sol

```diff
@@ -568,10 +600,11 @@ contract StrategyPrismaConvexFactoryClonable is BaseStrategy {
             return true;
         }
 
-        // harvest if we have a sufficient profit to claim and are max boosted.
+        // harvest if we have a sufficient profit to claim, are max boosted, and we're claiming rewards.
         if (
             claimableProfit > harvestProfitMinInUsdc &&
-            (claimsAreMaxBoosted() || forceClaimOnce)
+            _claimParams.shouldClaimRewards &&
```

Comment:
If the first two conditions are true, should it enter in [this](https://github.com/dudesahn/CurveVaultFactory/pull/10/files#diff-9cb489e35c778104431c910adc0a71bed9bc825ae89f7af8088ab57a56ba9ac3R587-R588) if?

Reply:
they're different; one is MAX profit and the other is MIN profit